Directory.Packages.props

```diff
@@ -87,7 +87,7 @@
         <PackageVersion Include="BitFaster.Caching" Version="2.2.0" />
         <PackageVersion Include="CliWrap" Version="3.6.4" />
         <PackageVersion Include="DynamicData" Version="7.14.2" />
-        <PackageVersion Include="GameFinder" Version="3.1.0" />
+        <PackageVersion Include="GameFinder" Version="3.2.0" />
```

Comment:
Version [`3.2.1`](https://github.com/erri120/GameFinder/releases/tag/v3.2.1) should be up on NuGet.